app/controllers/mentor/registrations_controller.rb

```diff
@@ -6,12 +6,23 @@ def index
   end
 
   def new
-    @tracks = Track.active.order(:title).map{|t|[t.title, t.id]}
+    load_tracks
+    @errors = []
   end
 
   def create
-    MakeUserAMentor.(current_user, params[:track_id])
-    redirect_to mentor_dashboard_path
+    command = MakeUserAMentor.new(current_user, params[:track_id])
+
+    command.call
```

Comment:
Maybe `command.()` for consistency's sake (I figure it'll be easier to search for when we add this monad style thing into the command patter on mandate).